circuit_knitting/cutting/qpd/qpd.py

```diff
@@ -546,19 +546,51 @@ def decompose_qpd_instructions(
     return new_qc
 
 
-_qpdbasis_from_gate_funcs: dict[str, Callable[[Instruction], QPDBasis]] = {}
+_qpdbasis_from_instruction_funcs: dict[str, Callable[[Instruction], QPDBasis]] = {}
 
 
-def _register_qpdbasis_from_gate(*args):
+def _register_qpdbasis_from_instruction(*args):
     def g(f):
         for name in args:
-            _qpdbasis_from_gate_funcs[name] = f
+            _qpdbasis_from_instruction_funcs[name] = f
         return f
 
     return g
 
 
-def qpdbasis_from_gate(gate: Instruction) -> QPDBasis:
+@deprecate_func(
+    since="0.3.0",
```

Comment:
Since we are only supporting minor releases right now, should we include the patch version anymore? I've been doing 0.4 in my PRs (I think).

Reply:
Regardless of whether we actually make patch-level releases, I think either way is acceptable.  Qiskit includes a mix of conventions, but uses `since=0.x.0` much more often than `since=0.x`.  Currently, CKT contains one `since="0.3"` and one `since="0.3.0"`.  I think it's fine either way.